hiredis/hiredis.pyi

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Callable, Optional, Union
+
+class HiredisError(Exception): ...
+class ProtocolError(HiredisError): ...
+class ReplyError(HiredisError): ...
+
+class Reader:
+    def __init__(
+        self,
+        protocolError: Callable[[str], Exception] = ...,
+        replyError: Callable[[str], Exception] = ...,
+        encoding: Optional[str] = ...,
+        errors: Optional[str] = ...,
+    ) -> None: ...
+    def feed(
+        self, __buf: Union[str, bytes], __off: int = ..., __len: int = ...
+    ) -> None: ...
+    def gets(self, __shouldDecode: bool = ...) -> Any: ...
```

Comment:
might be able to make this return type more precise, but that might also cause more issues for consumers of the library